src/main.html

```diff
@@ -164,6 +164,12 @@
                     <input id="port-override" type="text" value="/dev/rfcomm0"/>
                 </label>
             </div>
+            <div id="firmware-virtual-option">
```

Comment:
This is flickering on startup - can by fixed by making it hidden by default. Also, the label looks odd in linux:

![image](https://user-images.githubusercontent.com/4742747/102011516-88462d80-3da9-11eb-931d-262d33397c75.png)

Reply:
This is still flickering on startup.

Reply:
Sorry, on my end it doesn't flicker so I hope `display: none;` fixes it